docs/reference/core/ambassador.md

```diff
@@ -41,6 +41,7 @@ spec:
 | `server_name: envoy` | By default Envoy sets server_name response header to `envoy`. Override it with this variable |  |
 | `service_port: 8080` | If present, service_port will be the port Ambassador listens on for microservice access. If not present, Ambassador will use 8443 if TLS is configured, 8080 otherwise. |  |
 | `statsd` | Configures Ambassador statistics. These values can be set in the Ambassador module or in an environment variable. For more information, see the [Statistics reference](../../statistics#exposing-statistics-via-statsd). |  |
+| `upstream_ambassador_namespace` | Controls whether Ambassador will resolve upstream services assuming they are in the same namespace as the element referring to them, e.g. a Mapping in namespace `foo` will look for its service in namespace `foo`. If `true`, Ambassador will resolve the upstream services assuming they are in the same namespace as Ambassador, unless the service explicitly mentions a different namespace. | `upstream_ambassador_namespace: false` |
```

Comment:
This name sounds to me like the value would be a namespace not a boolean.

Reply:
I propose using `use_upstream_ambassador_namespace` as this would also fit the pattern of other Module configuration options.

Reply:
How about `use_ambassador_namespace_for_service_resolution` just to be clear that you're not talking about "upstream ambassadors".